specifyweb/frontend/js_src/lib/components/FormParse/index.ts

```diff
@@ -468,6 +468,8 @@ export type FormCondition =
   | State<'Always'>
   | undefined;
 
+export const EMPTY_VALUE_CONDITION = '_EMPTY';
```

Comment:
could you clarify why this is needed, and why having an empty string `""` isn't sufficient?

Reply:
Just so users have a clearer way of doing an empty value check. A condition like `<rows condition="childcog=">` will work for an empty value check but it reads like an incomplete condition and on first glance doesn't seem like it would be doing anything at all. The `_EMPTY` keyword should just make things clearer for users